### PR TITLE
Update replicaset.md for Issue #12205 with k8s.io/docs/concepts/workloads/controllers/replicaset/ 

### PR DESCRIPTION
**Pull request by SupriyaSirbi**
Overview : This pull request is for Issue #12205 which updates the replicaset.md file

When ReplicaSets have the same label selectors but different metadata, the Replicas are created by controller because each Replicaset ignores the Pods created by another.

### CODE REVIEW

**Comment by SupriyaSirbi** [comment]
Hello @tengqm Could you please review and approve the PR?

**Comment by tengqm** [comment]
@SupriyaSirbi Sorry I have lost my local k8s env for testing this behavior. Better leave it to others for review.

**Comment by SupriyaSirbi** [comment]
@madhusudancs and @Kashomon : Please can you review this PR?

**Comment by Kashomon** [comment]
Probably not a good person. I haven't done website stuff for a while.

**Comment by SupriyaSirbi** [comment]
/assign @chenopis 
/assign @jimangel 

Hello Both,

Could you please review the PR for the same?

Thanks,
Supriya

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ created the other pods. Kubernetes does not stop you from doing this.
 If you do end up with multiple controllers that have overlapping selectors, you
 will have to manage the deletion yourself.
 
+{{< note >}}
+For 2 ReplicaSet specifying the same `.spec.selector` but different `.spec.template.metadata.labels` and `.spec.template.spec`, each ReplicaSet ignores the Pods created by another and controller creates the Replicas.
```
Comment (zparnold) [inline content/en/docs/concepts/workloads/controllers/replicaset.md]:
```suggestion
For 2 ReplicaSets specifying the same `.spec.selector` but different `.spec.template.metadata.labels` and `.spec.template.spec` fields, each ReplicaSet ignores the Pods created by the other ReplicaSet.
```

**Comment by zparnold** [comment]
@SupriyaSirbi PTAL I changed the language here a bit. Could you also fix the merge conflict? Then we can get this merged!

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 1 above)
Comment (SupriyaSirbi) [reply content/en/docs/concepts/workloads/controllers/replicaset.md]: Made the changes and requested review

**Comment by SupriyaSirbi** [comment]
Created a new pull request for the same

**Pull request closed without merging**